services/galley/test/integration/API/MLS.hs

```diff
@@ -1102,6 +1107,28 @@ testAppMessage2 = do
         WS.assertMatchN_ (5 # WS.Second) wss $
           wsAssertMLSMessage conversation bob (mpMessage message)
 
+testAppMessageUnreachable :: TestM ()
+testAppMessageUnreachable = do
```

Comment:
This test case is OK, though I'd also appreciate if you could add a mixed outcome test case, namely one where the message can be sent to some, but not all remote backends.